addon-test-support/helpers.js

```diff
@@ -0,0 +1,7 @@
+import { getContext } from '@ember/test-helpers';
```

Comment:
FWIW, you can change the import path to `import { getContext } from `ember-test-helpers’` and have support going waaaaaay back. That module is shipped still, and `getContext` has existed for quite some time. The only caveat is that `.owner` won’t exist in older style tests (there you would need `.container` probably?).

I don’t feel strongly at all here, just pointing out some options...

Reply:
I think we can leave it this way and tell users this only works with the new tests, and perhaps use it as a carrot for people to update.

Reply:
👍